worlds/sm64ex/__init__.py

```diff
@@ -54,10 +54,10 @@ def set_rules(self):
         set_rules(self.world, self.player, self.area_connections)
         if self.topology_present:
             # Write area_connections to spoiler log
-            for painting_id, course_id in self.area_connections.items():
+            for painting_id, destination in self.area_connections.items():
                 self.world.spoiler.set_entrance(
-                    sm64courses[painting_id] + " Painting",
-                    sm64courses[course_id],
+                    sm64paintings[painting_id] + " Painting",
+                    sm64courses[int(destination/10)],
```

Comment:
"Integer Division" is // in python, for example:
24 // 10 -> 2

Reply:
Thats a cool trick! Amended